mkFit/PropagationMPlex.cc

```diff
@@ -668,14 +668,14 @@ void helixAtZ(const MPlexLV& inPar,  const MPlexQI& inChg, const MPlexQF &msZ,
 		<< " pxin=" << pxin << " pyin=" << pyin);
 
       const float pxcaMpysa = pxin*cosa - pyin*sina;
-      errorProp(n,0,2) = -tanT * ipt * pxcaMpysa;;
-      errorProp(n,0,3) = 2*k*pt*pt*sinah*(sinP*sinah - cosP*cosah) + deltaZ*tanT*pxcaMpysa;
+      errorProp(n,0,2) = -tanT * ipt * pxcaMpysa;
+      errorProp(n,0,3) = k*pt*pt*( cosP*(alpha*cosa - sina) + sinP*( 1.f - cosa - alpha*sina) );
```

Comment:
please regroup the second factor to use alpha/2
`1.f - cosa - alpha*sina = 2.f*sinah*sinah - alpha*2.f*sinah*cosah = 2.f*sinah*(sinah - alpha*cosah)`
this will avoid `1 -1` kind of subtraction